src/handle_action.cpp

```diff
@@ -110,10 +110,11 @@ static const bionic_id bio_remote( "bio_remote" );
 
 static const trait_id trait_HIBERNATE( "HIBERNATE" );
 static const trait_id trait_PROF_CHURL( "PROF_CHURL" );
-static const trait_id trait_PROF_HELI_PILOT( "PROF_HELI_PILOT" );
 static const trait_id trait_SHELL2( "SHELL2" );
 static const trait_id trait_WAYFARER( "WAYFARER" );
 
+static const proficiency_id proficiency_prof_helicopter_pilot( "prof_helicopter_pilot" );
```

Comment:
I think `proficiency_helicopter_pilot` or maybe even just `prof_helicopter_pilot` is sufficient.

Reply:
It was initially the first, but that was making clang-tidy complain.

Reply:
If you'd prefer the shorter `prof_helicopter_pilot` to be acceptable to clang-tidy, you can update the `HardcodedPrefixes` map in `tools/clang-tidy-plugin/StaticStringIdConstantsCheck.cpp`, adding `"proficiency": ""`.